<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crear_grupo</class>
 <widget class="QDialog" name="crear_grupo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Grupo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nuevo Grupo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Descripci√≥n:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtDescripcion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCrear">
       <property name="text">
        <string>Crear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtDescripcion</tabstop>
  <tabstop>btCrear</tabstop>
  <tabstop>btCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btCancelar</sender>
   <signal>clicked()</signal>
   <receiver>crear_grupo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
